Matlab/Feature Extraction/extract_Hog_Features.m

```diff
@@ -0,0 +1,11 @@
+function [featureVectors] = extract_Hog_Featues(images, imY, imX)

```

Comment:
(repeat for each of the following files)
Was this Jesus' code, or yours? We need to keep track of which files we wrote and which we didn't.